packages/react-tabs/src/components/Tab/Tab.types.ts

```diff
@@ -0,0 +1,59 @@
+import type { ComponentProps, ComponentState, IntrinsicShorthandProps } from '@fluentui/react-utilities';
+
+/**
+ * Any value that identifies a specific tab.
+ */
+export type TabValue = unknown;
+
+export type TabSlots = {
+  /**
+   * Root of the component.
+   */
+  root: IntrinsicShorthandProps<'div'>;
+
+  /**
+   * Icon that renders before the content.
+   */
+  icon?: IntrinsicShorthandProps<'span'>;
+
+  /**
+   * Component children are placed in this slot
+   * Avoid using the `children` property in this slot in favour of Component children whenever possible.
+   */
+  content: IntrinsicShorthandProps<'span'>;
```

Comment:
The padding and gaps between medium/small, horizontal/vertical, and content-only/icon-only/content-and-icon would require a whole bunch of conditional styling with very odd spacing values.  We went through several rounds with design and landed on these spacing off the content to have them add up to something sensical that made the style hook readable by humans. :-D 

I can postulate a PR without the content slot if you think looking at it would help.  I've added and remove the content slot about three times now :-S